bot/animeParser.js

```diff
@@ -3,26 +3,51 @@ const client = new Discord.Client();
 const config = require('../config.json');
 const sharedFunc = require("../sharedFunc");
 const {DateTime} = require("luxon");
+const {MongoClient} = require('mongodb');
 
 let subreddit;
 
+function checkMongo() {
+    const guild = client.guilds.get("");
+}
+
 /**
  * Client listener that runs exactly one (1) time when the bot first starts. Anything that needs to be ran to set up
  * bot functionality should be run in this function. Currently it is used to signal in console that the bot has started
  * and the create the reference to the AnimeCalendar subreddit.
+ *
+ * @posts: Fetches all the possible images for that day and chooses one at random to display
+ *
+ * @setInterval: After 24 hours, the bot will reach out to MongoDB and ask what servers it should be sending a daily AniDay image too
  */
 client.on('ready', async x => {
     console.log("i'm lit on " + client.guilds.cache.size + " servers.");
+
+    let posts = await sharedFunc.getGoogle(DateTime.local().toLocaleString({month: 'short', day: '2-digit'}));
+    var randomIMG = posts[Math.floor(Math.random() * posts.length)];
+    let dailyGuildArray = await sharedFunc.dailyMongoSender();
```

Comment:
Shouldn't this be inside the setinterval, so that the daily server list is updated here.

Reply:
Good point, I didn't consider that. Thanks.